utils/test-utils/build.gradle.kts

```diff
@@ -12,13 +12,15 @@ dependencies {
 
     api(libs.edc.junit)
     api(libs.awaitility.java)
+    api(libs.postgres)
     api(project(":extensions:wrapper:clients:java-client"))
     api(project(":utils:json-and-jsonld-utils"))
 
     implementation(project(":extensions:policy-always-true"))
     implementation(project(":utils:versions"))
     implementation(libs.edc.jsonLdSpi)
     implementation(libs.edc.jsonLd)
+    implementation(libs.edc.runtimeMetamodel)
```

Comment:
do really ALL the modules need this?

Reply:
all the module that use an EDC dependency, directly or transitively.